### DISCUSSION TITLE
Setting Y-axis scaling on chart

### DISCUSSION THREAD

**Discussion by byeugenius** [klinecharts/KLineChart#564] [Q&A]
How can i change Y-axis scaling on my chart without mouse, but with code in my project?
For example, we can change X-axis scaling with `zoomAtCoordinate()`.

**Comment by char101** [comment]
Use `setRange`

https://github.com/klinecharts/KLineChart/blob/045973502f210bd92c060c0ac0cb136d64777106/src/Event.ts#L258
